eventuals/task.h

```diff
@@ -238,34 +254,56 @@ struct _TaskFromToWith {
     K_ k_;
```

Comment:
Should these member variables be private?

If they're intentionally public, they probably deserve comments.

Reply:
Will refactor in another commit